docs/userguide/share_data.rst

```diff
@@ -65,3 +65,34 @@ Data copied from file explorer also appear in the Parsec client. In addition, th
     Parsec client can work while offline (however only data present locally
     are available), synchronization will occur automatically as soon as the
     connection with the server is established.
+
+Bulk assign workspace roles to users
+------------------------------------
+
+Assigning the same roles to multiple users on multiple workspaces can be a bit annoying. Parsec provides a way to bulk assign the roles of one user to another.
```

Comment:
I'd use "tedious" instead of "annoying":
```suggestion
Manually assigning the same workspace roles to multiple users can be a bit tedious and error prone. Parsec provides a way to bulk assign the roles from one user to another.
```
In French:
```
L'assignation manuelle des mêmes rôles d'espace de travail à plusieurs utilisateurs peut s'avérer fastidieuse et source d'erreurs. Parsec propose un moyen d'attribuer en bloc les rôles d'un utilisateur à un autre.
```